deepinv/datasets/lidc_idri.py

```diff
@@ -0,0 +1,139 @@
+from typing import (
+    Any,
+    Callable,
+    NamedTuple,
+    Optional,
+)
+import os
+from pydicom import dcmread
+import torch
+import numpy as np
+import pandas as pd
+
+
+class LidcIdriSliceDataset(torch.utils.data.Dataset):
+    """Dataset for `LIDC-IDRI <https://www.cancerimagingarchive.net/collection/lidc-idri/>`_ that provides access to CT image slices.
+
+    | The Lung Image Database Consortium image collection (LIDC-IDRI) consists
+    | of diagnostic and lung cancer screening thoracic computed tomography (CT)
+    | scans with marked-up annotated lesions.
+    | It is a web-accessible international resource for development, training,
```

Comment:
I would remove this paragraph "It is a web-accessible international resource for dev ..."

since the usage in the library is mostly focused on reconstructing the CT scans